common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -19,6 +19,7 @@ if Backbone?
     initialize: (options) ->
       super()
       @mode = options.mode or "inline"  # allowed values are "tab" or "inline"
+      @context = options.context or "course"  # allowed values are "course" or "standalone"
```

Comment:
DiscussionModuleView is the outermost view, and it makes a DiscussionThreadView per post (the terminology is very confusing). I needed to wire the context down through the children so that it can get to the DiscussionThreadEditView (the edit modal).